src/gen/common/v1/common_pb.d.ts

```diff
@@ -0,0 +1,474 @@
+// package: viam.common.v1
```

Comment:
can we avoid committing all generated files since the publish will do that work for us?

Reply:
yeah - we should keep track of the files `main.ts` references directly (so just the `esm` files that rollup generates) and ignore the rest

Reply:
we dont even need to commit that. no one clones this to pull those in. they can use the npm publish